src/index.js

```diff
@@ -0,0 +1,177 @@
+    console.log('tempThemeToggle called');
+
+    addTemp();
+    lowerTemp();
+
+    const tempAdjustElement = document.getElementById("tempAdjust");
+    const tempAdjustLandscape = document.getElementById("landScape");
+
+    if (state.temp >= 32){
+        tempAdjustElement.className = "tooHot";
+        tempAdjustLandscape.className = "landHot";
+    } else if (state.temp >= 25 && state.temp < 32){
+        tempAdjustElement.className = "hot";
+        tempAdjustLandscape.className = "landTemperate";
+    } else if (state.temp > 0 && state.temp <= 11) {
+        tempAdjustElement.className = "cold";
+        tempAdjustLandscape.className = "landCold";
+    } else if (state.temp <= 0){
+        tempAdjustElement.className = "freezing";
+        tempAdjustLandscape.className = "landFreezing";
+    };
+};
+
+const skyToggle = () => {
+    // console.log('skyToggle called');
+
+    const skySelectElement = document.getElementById("sky-id");
+    const weatherLandscape = document.getElementById("sky");
+
+    if (skySelectElement.value === "sunny") {
+        weatherLandscape.className = "skySunny";
+        console.log("SUNNY")
+    } else if (skySelectElement.value === "partlyCloudy") {
+        weatherLandscape.className = "skyPartlyCloudy";
+    } else if (skySelectElement.value === "overcast") {
+        weatherLandscape.className = "skyOvercast";
+    } else if (skySelectElement.value === "rainy") {
+        weatherLandscape.className = "skyRainy";
+    } else if (skySelectElement.value === "snow"){
+        weatherLandscape.className = "skySnow";
+    };
+};
+// create a skyToggle function that changes the images of the sky
+// it will be almost the same as tempTheme, getting element id #sky
+// and then if one of the buttons from select_sky function is called
+// if that list element (doc.getElementbyId) sky-id 
+// is equal to rain change className to skyRainy
+// if that list elelment sky-id is equal to snow
+// change className to skySnow
+// if sky-id is equal to partyCloudy, change classname to skyPartlyCloudy
+// if sky-id is equal to sunny, change className to skySunny
+
+
+const cityNameResponse = () => {
+
+    const inputCity = document.getElementById('cityText').value;
+    const cityDisplay = document.getElementById('city');
+    state.city = inputCity
+    cityDisplay.textContent = state.city;
+};
+
+const getLatAndLong = () => {
+    axios
+    .get('http://localhost:5000/location',{
+    params: {
+    q: state.city,
+    },
+    }).then((response) => {
+    state.lat = response.data[0]['lat'];
+    state.lon = response.data[0]['lon'];
+    getWeather();
+    })
+    .catch(err => console.log('error finding latitude and longitude', {err}));
+};
+
+const getIRLTemp = () => {
+    state.city = document.getElementById("cityText").value;
+    getLatAndLong();
+};
+
+const tempKtoC = (temp) => {
+    return (temp - 273.15);
+};
+
+const tempKtoF = (temp) => {
+    return (temp-273.15) * (9/5) + 32;
+}
```

Comment:
If this function isn't used, we should remove it. If it's been committed, if we decide we need it later, we can always recover it by looking through our git history.